repo/manifest/manifest_manager_test.go

```diff
@@ -381,3 +395,34 @@ func TestManifestAutoCompaction(t *testing.T) {
 		require.NoError(t, mgr.b.Flush(ctx))
 	}
 }
+
+func TestManifestAutoCompactionWithReadOnly(t *testing.T) {
+	ctx := testlogging.Context(t)
+	data := blobtesting.DataMap{}
+
+	bm := newContentManagerForTesting(ctx, t, data, false)
```

Comment:
nit: is there a way of making the `readOnly` parameter more explicit?

Either using a named `const`, or maybe changing the function signature to take an options struct or similar. Thoughts?